administration/package.json

```diff
@@ -8,13 +8,21 @@
     "@blueprintjs/datetime": "^4.3.12",
     "@blueprintjs/popover2": "^1.7.3",
     "@blueprintjs/select": "^4.6.6",
+    "@emotion/react": "^11.10.4",
+    "@emotion/styled": "^11.10.4",
+    "@fontsource/roboto": "^4.5.8",
+    "@mui/icons-material": "^5.10.9",
+    "@mui/lab": "^5.0.0-alpha.104",
+    "@mui/material": "^5.10.9",
```

Comment:
Haven't reviewed the rest of the PR: Do we need Material? Isn't blueprint enough?

I made pleasant experience with that UI framework: https://chakra-ui.com/

Reply:
Blueprint is unfortunately focused on desktop environments. @mui was the next best thing that came to mind with good mobile support.